1657407 Add descriptions for SEO. (#17372)
dotnet/docs: docs/framework/install/guide-for-developers.md

---
title: "Install the .NET Framework developer pack or redistributable"

> description: "Developers can install the .NET Framework or install it with their applications. You can install Developer Pack or targeting pack versions and redistributables."

Reviewer: Maybe something like this?

```suggestion
description: "Developers can download and install the .NET Framework, a developer pack, and a targeting pack. Developers can also include the .NET Framework redistributable with their apps."
```

Author: This is about 10 char over the 160 limit for descriptions. 

Maybe: 

Developers can download and install the .NET Framework developer pack and targeting pack. You can include the .NET Framework redistributable with your apps.

Author: @thraka - let me know if my new wording is OK.

Reviewer: Sounds good.
